.github/CODEOWNERS

```diff
@@ -49,5 +49,5 @@ crates/wash-cli @wasmCloud/wash-maintainers
 crates/wash-lib @wasmCloud/wash-maintainers
 examples @wasmCloud/examples-maintainers
 vendor/capability-providers @wasmCloud/capability-provider-maintainers
-typescript @wasmCloud/washboard-maintainers
+examples/typescript @wasmCloud/typescript-maintainers
```

Comment:
This team [exists](https://github.com/orgs/wasmCloud/teams/typescript-maintainers), but may not have access to the repo.